app/views/dashboard/ng-partials/widgets/_hardware.html.haml

```diff
@@ -69,26 +69,26 @@
                 %td
                   %label INVERT ENDPOINTS
                 %td
-                  %calibrationbutton{toggleval: "MOVEMENT_INVERT_ENDPOINTS_X"}
+                  %calibrationbutton.left{toggleval: "MOVEMENT_INVERT_ENDPOINTS_X"}
                 %td
-                  %calibrationbutton{toggleval: "MOVEMENT_INVERT_ENDPOINTS_Y"}
+                  %calibrationbutton.left{toggleval: "MOVEMENT_INVERT_ENDPOINTS_Y"}
                 %td
-                  %calibrationbutton{toggleval: "MOVEMENT_INVERT_ENDPOINTS_Z"}
+                  %calibrationbutton.left{toggleval: "MOVEMENT_INVERT_ENDPOINTS_Z"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [83/80]